app/code/Magento/GoogleGtag/Block/Ga.php

```diff
@@ -0,0 +1,198 @@
+     * @param array $data
+     * @param Cookie $cookieHelper
+     */
+    public function __construct(
+        Context $context,
+        CollectionFactory $salesOrderCollection,
+        GtagConfiguration $googleGtagConfig,
+        Cookie $cookieHelper,
+        array $data = []
+    ) {
+        $this->googleGtagConfig = $googleGtagConfig;
+        $this->salesOrderCollection = $salesOrderCollection;
+        $this->cookieHelper = $cookieHelper;
+        parent::__construct($context, $data);
+    }
+
+    /**
+     * Get config
+     *
+     * @param string $path
+     * @return mixed
+     */
+    public function getConfig($path): string
+    {
+        return $this->_scopeConfig->getValue($path, ScopeInterface::SCOPE_STORE);
+    }
+
+    /**
+     * Get helper
+     *
+     * @return GtagConfiguration
+     */
+    public function getHelper(): GtagConfiguration
+    {
+        return $this->googleGtagConfig;
+    }
+
+    /**
+     * Get a specific page name (may be customized via layout)
+     *
+     * @return string|null
+     */
+    public function getPageName(): ?string
+    {
+        return $this->_getData('page_name');
+    }
+
+    /**
+     * Render GA tracking scripts
+     *
+     * @return string
+     */
+    protected function _toHtml()
+    {
+        if (!$this->googleGtagConfig->isGoogleAnalyticsAvailable()) {
+            return '';
+        }
+
+        return parent::_toHtml();
+    }
+
+    /**
+     * Return cookie restriction mode value.
+     *
+     * @return bool
+     */
+    public function isCookieRestrictionModeEnabled(): bool
+    {
+        return (bool) $this->cookieHelper->isCookieRestrictionModeEnabled();
+    }
+
+    /**
+     * Return current website id.
+     *
+     * @return int
+     */
+    public function getCurrentWebsiteId(): int
+    {
+        return (int) $this->_storeManager->getWebsite()->getId();
+    }
+
+    /**
+     * Return information about page for GA tracking
+     *
+     * @link https://developers.google.com/analytics/devguides/collection/gtagjs
+     * @link https://developers.google.com/analytics/devguides/collection/ga4
+     *
+     * @param string $measurementId
+     * @return array
+     */
+    public function getPageTrackingData($measurementId): array
+    {
+        return [
+            'optPageUrl' => $this->getOptPageUrl(),
+            'measurementId' => $this->escapeHtmlAttr($measurementId, false)
+        ];
+    }
+
+    /**
+     * Return information about order and items for GA tracking.
+     *
+     * @link https://developers.google.com/analytics/devguides/collection/ga4/ecommerce#purchase
+     * @link https://developers.google.com/gtagjs/reference/ga4-events#purchase
+     * @link https://developers.google.com/analytics/devguides/collection/gtagjs/enhanced-ecommerce#product-data
+     * @link https://developers.google.com/gtagjs/reference/event#purchase
+     *
+     * @return array
+     * @since 100.2.0
```

Comment:
Why is this specific method is marked with `@since` and how is it exactly `100.2.0`?

Reply:
Removed this code, as its a new Module